examples/web-form-with-express/public/client.js

```diff
@@ -0,0 +1,136 @@
+// This file is run by the browser each time your view template is loaded
+
+/**
+ * Define variables that reference elements included in /views/index.html:
+ */
+
+// Forms
+const dbForm = document.getElementById("databaseForm")
+const pageForm = document.getElementById("pageForm")
+const blocksForm = document.getElementById("blocksForm")
+const commentForm = document.getElementById("commentForm")
+// Table cells where API responses will be appended
+const dbResponse = document.getElementById("dbResponse")
+const pageResponse = document.getElementById("pageResponse")
+const blocksResponse = document.getElementById("blocksResponse")
+const commentResponse = document.getElementById("commentResponse")
+
+/**
+ * Functions to handle appending new content to /views/index.html
+ */
+
+// Appends the API response to the UI
+const appendApiResponse = function (res, el) {
+  console.log(res)
```

Comment:
q -- was this console.log meant to be left in? more than okay if so, just pointing out in case it wasn't

same with other console.log's in this file

Reply:
In this case yes, just so they could see the full response (including an error). A few pieces of data are appended to the UI but I left the console logs in in case it's useful to the person testing it out to see the full response.